demo/src/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { Buffer } from 'buffer';
+import { FC, useCallback, useState } from 'react';
+import { createRoot } from 'react-dom/client';
+import HandleClient, { HandleClientContext, HandleClientProvider, KoraLabsProvider } from '../../lib';
+import { BlockfrostHandle, BlockfrostProvider } from '../../lib/classes/providers/BlockfrostProvider.class';
+
+import { IHandle } from '@koralabs/handles-public-api-interfaces';
+import JSONView from 'react-json-view';
+
+const KoraInstance = new HandleClient({
+    context: HandleClientContext.MAINNET,
+    provider: new KoraLabsProvider(HandleClientContext.MAINNET)
+});
+
+const BlockfrostInstance = new HandleClient({
+    context: HandleClientContext.MAINNET,
+    // @ts-ignore The BLOCKFROST_API_KEY gets overwritten in webpack config.
+    provider: new BlockfrostProvider(HandleClientContext.MAINNET, BLOCKFROST_PROJECT_ID)
+});
+
+const App: FC = () => {
+    const [handleName, setHandleName] = useState('');
+    const [cip68, setCip68] = useState(false);
+    const [blockfrostResult, setBlockfrostResult] = useState<BlockfrostHandle | IHandle>();
+    const [koraResult, setKoraResult] = useState<IHandle>();
+    const [loading, setLoading] = useState(false);
+
+    const handleSearch = useCallback(
+        async (handle: string) => {
+            setLoading(true);
+
+            const realHandle = (cip68 ? HandleClientProvider.CIP68_PREFIX : '') + Buffer.from(handle).toString('hex');
```

Comment:
Now your question about prefixes makes way more sense. You can use this package if you like. https://github.com/koralabs/handles-public-api-interfaces/blob/master/index.ts#L19